src/scripts/OSFramework/OSUI/Event/DOMEvents/Listeners/AbstractListener.ts

```diff
@@ -16,12 +16,34 @@ namespace OSFramework.OSUI.Event.DOMEvents.Listeners {
 		private _eventTarget: HTMLElement;
 		// Store the listener type
 		private _eventType: GlobalEnum.HTMLEvent;
-		// Store the listener callback
+
+		/**
+		 * Store the listener callback
+		 *
+		 * @protected
+		 * @type {EventListenerObject}
+		 * @memberof AbstractListener
+		 */
 		protected eventCallback: EventListenerObject;
 
-		constructor(eventTarget, eventType) {
+		/**
+		 * Flag to indicate if event will be dispatched to the registered listener before being dispatched to any EventTarget beneath it in the DOM tree.
+		 *
+		 * @protected
+		 * @memberof AbstractListener
+		 */
+		protected useCapture = false;
+
+		/**
+		 * Creates an instance of AbstractListener.
+		 *
+		 * @param {unknown} eventTarget, window can be in use, so, HTMLElement can't be the type of the input by default
+		 * @param {GlobalEnum.HTMLEvent} eventType
+		 * @memberof AbstractListener
+		 */
+		constructor(eventTarget: unknown, eventType: GlobalEnum.HTMLEvent) {
```

Comment:
Why the type `unknown`? After all, 2 lines after we're doing the "cast"...

Reply:
Done!
Thanks for the heads up mate!